kaizen/agent_runners.py

```
@@ -0,0 +1,497 @@
+"""
+Agent registry and base runner implementation.
+"""
+from abc import ABC, abstractmethod
+from typing import Dict, Any, Type, Optional, List, Union, TypeVar, Callable, Sequence, Mapping, Set, FrozenSet, Tuple
```

Ruff (0.11.9)

`typing.Union` imported but unused

Remove unused import

(F401)